docs/src/main/asciidoc/_galleon/Galleon_layers.adoc

```diff
@@ -1,6 +1,15 @@
 [[wildfly_layers]]
 === WildFly Layers
 
+A Galleon layer is a name that identifies a server capability (e.g.: jaxrs, 
+ejb, microprofile-config, jpa) or an aggregation of such capabilities. A layer captures a server capability in the form of:
+
+* A piece of server XML configuration (e.g.: extension, configured subsystem, interfaces) that describes the capability.
+* A set of JBoss modules that implements the capability.
```

Comment:
s/JBoss modules/JBoss Modules modules/g

or just "modules".

More words, but perhaps

* A set of modules and other filesystem content that implements the capability.

Maybe that's too much information. Layers bring things other than modules is the point; e.g. core-tools, vault-tools etc.

Reply:
@bstansberry , I focused on the modules/configs. I added what you suggested. I think it is vague enough to cover the "other files".